src/FillCommand.php

```diff
@@ -7,7 +7,9 @@
 
 class FillCommand extends Command
 {
-    protected $signature = 'ksql:fill {resourceName?}';
+    protected $signature = 'ksql:fill
+                            {resourceName? : Optional specific resource to fill}
+                            {resourceIds?* : Optional list of specific resource ids to fill}';
```

Comment:
I used the built in support for array parameters but the syntax is a little clunky IMO... `artisan ksql:fill merchant 03NyX02V015XMrkQFT1iFbG 02RGwAdnXnoxI2rPZTySLqq` vs something like `artisan ksql:fill merchant 03NyX02V015XMrkQFT1iFbG,02RGwAdnXnoxI2rPZTySLqq`